ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -782,6 +782,10 @@ def run_ocs_upgrade(
         # in pending state
         is_all_csvs_succeeded = check_all_csvs_are_succeeded(namespace=namespace)
         assert is_all_csvs_succeeded, "Not all CSV's are in succeeded state"
+        if config.ENV_DATA.get("is_multus_enabled"):
```

Comment:
shouldn't we also check if the target cluster version is 4.16?

Reply:
we are working with branches so we dont need to add condition for ODF version

Reply:
but this will remain the same also in 4.17, where this change won't be required

Reply:
I added a configuration file. because we need to test it on ODF4.15->4.16 and on ODF4.16-4.17:

Test case 1:
Deploy OCP4.15+ODF4.15+Multus  [cluster-net+public_net, cluster_net_only, public_net_only] 
upgrade OCP4.15->16
upgrade ODF4.15->16 [holder pods exist but we don't delete them]
upgrade OCP4.16->17
upgrade ODF4.16-> 17
Run " forcefully disable holder pods"
Test case 2:
Deploy OCP4.15+ODF4.15+Multus [cluster-net+public_net, cluster_net_only, public_net_only] 
upgrade OCP4.15->16
upgrade ODF4.15->16
Run " forcefully disable holder pods"
upgrade OCP4.16->17
upgrade ODF4.16-> 17 [holder pods do not exist]